_posts/0000-01-02-Kaleem-ch.md

```diff
@@ -0,0 +1 @@
+adding new file in the repository
```

Comment:
---
layout: slide
title: "Welcome to our second slide!"
---
Kaleem's
new comment